hub/migrations/0012_alter_constance_config.py

```diff
@@ -0,0 +1,44 @@
+import json
+
+from constance import config
+from django.db import migrations, models
+
+
+def alter_constance_config(apps, schema_editor):
+    # Check that the USER_METADATA_FIELDS has `full_name` set
+    user_metadata = json.loads(getattr(config, 'USER_METADATA_FIELDS'))
```

Comment:
We do not use serialized json in `base.py` but `LazyJSONSerializable` objects. Please use `to_python_objects` instead. 
See [this comment](https://github.com/kobotoolbox/kpi/pull/4569/files#r1290665109) 

Why are you using `getattr` here? Can't you call `config.USER_METADATA_FIELDS` directly? If attribute did not exist, it would return `None`, right? So, this line would raise an error.